egg/zoo/compo_vs_generalization/data.py

```diff
@@ -64,6 +64,7 @@ def select_subset_V2(data, n_subset, n_attributes, n_values, random_seed=7):
 
 
 def one_hotify(data, n_attributes, n_values):
+    # data: list of tuples of length n_attributes
```

Comment:
Make this a docstring (`"""`), ideally with a description quick description of what the function is doing. You might also add type annotation:

```python
def one_hotify(data: List, n_attributes: int, n_values: int):
```